__tests__/app.test.js

```diff
@@ -177,3 +176,46 @@ describe("GET /api/articles/:article_id/comments", () => {
     expect(body.msg).toBe("Bad request");
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("Status: 201 responds with the comment that was posted", async () => {
+    const newComment = {
+      author: "icellusedkars",
+      body: "make way, king of code coming through!",
+    };
+    const { body: comment } = await request(app)
+      .post("/api/articles/8/comments")
+      .send(newComment)
+      .expect(201);
+
+    expect(comment.author).toEqual(expect.any(String));
+    expect(comment.body).toEqual(expect.any(String));
+    expect(comment.comment_id).toEqual(expect.any(Number));
+    expect(comment.article_id).toEqual(expect.any(Number));
+    expect(comment.votes).toEqual(expect.any(Number));
+    expect(comment.created_at).toEqual(expect.any(String));
+  });
+  test("Status: 404 responds with error when a user tries to post a comment to a non-existing article_id", async () => {
+    const newComment = {
+      author: "icellusedkars",
+      body: "make way, king of code coming through!",
+    };
+    const { body } = await request(app)
+      .post("/api/articles/200/comments")
+      .send(newComment)
+      .expect(404);
+
+    expect(body.msg).toBe("article does not exist");
+  });
+  test("Status 400 responds with error when a user tries to post a comment to an invalid article_id", async () => {
+    const newComment = {
+      author: "icellusedkars",
+      body: "make way, king of code coming through!",
+    };
+    const { body } = await request(app)
+      .post("/api/articles/eight/comments")
+      .send(newComment)
+      .expect(400);
+    expect(body.msg).toBe("Bad request");
+  });
+});
```

Comment:
I would also test for when the author does not exist in the database (404) and for when the required properties in the post body are not existing (400).